### ISSUE TITLE
Field <form_name>_complete is not included in auto generated transformation rules

### ISSUE THREAD

**Issue by targetpharma** [IUREDCap/redcap-etl-module#6]
REDCap generates a dropdown field for every form using the naming convention `<form_name>_complete`. This field is available in the data export, but is not explicitly listed in the project metadata. Therefore, the auto generated transformation rules do not include this field.

I made a quick addition to the `generateFields()` function in `RulesGenerator.php` as a proof of concept. But I don't have a pull request ready because I have not executed or modified any of the tests in the repository.

FWIW, my quick add to RulesGenerator.php, but there might be a better place or way to do this:

```
protected function generateFields($formName)
{
...
        // add the REDCap generated <form_name>_complete field
        $field['field_name'] = $formName .'_complete';
        $field['field_type'] = 'dropdown';
        $field['text_validation_type_or_show_slider_number'] = '';
        $rule = $this->getFieldRule($field);
        if (isset($rule)) {
            $fields .= $rule;
        }

        return $fields;
}
```

**Comment by mullen2** [comment]
Hi targetpharma,

It looks like my settings were not how they should have been, and I just saw this issue and your other one. Thanks for your feedback. We'll see what we can do.

Jim

**Comment by mullen2** [comment]
I have added an option for the REDCap-ETL application to generate form complete fields (https://github.com/IUREDCap/redcap-etl/releases/tag/1.1.0). I will try to add this as an option to the REDCap-ETL external module for the next release.

**Comment by mullen2** [comment]
An option to include the form complete fields when transformation rules are auto-generated has been added to the development version of the REDCap-ETL external module, and will be available in the next release, which is being worked on currently.

**Comment by mullen2** [comment]
This feature has been released in version 1.1.0
